__tests__/ros-ci.test.ts

```diff
@@ -39,10 +39,11 @@ describe("validate distribution test", () => {
 		expect(actionRosCi.validateDistros("", "eloquent")).toBe(true);
 		expect(actionRosCi.validateDistros("", "foxy")).toBe(true);
 		expect(actionRosCi.validateDistros("", "galactic")).toBe(true);
+		expect(actionRosCi.validateDistros("", "humble")).toBe(true);
 		expect(actionRosCi.validateDistros("", "rolling")).toBe(true);
 
 		expect(actionRosCi.validateDistros("noetic", "rolling")).toBe(true);
-		expect(actionRosCi.validateDistros("melodic", "dashing")).toBe(true);
+		expect(actionRosCi.validateDistros("melodic", "humble")).toBe(true);
```

Comment:
This change does not seem required, is there a reason for this change?

Reply:
Thought I'd just replace it since it's an old distro. I added another test case instead, even if it's a bit redundant: 1392af7e494cfe16b593b388b32c89f25b137761